docs/KNOWN_ISSUES.md

```diff
@@ -37,8 +51,8 @@ element further down the line :
 
 If you *must* call `player.reset()` then you'll have to manually restart the `videojs-contrib-eme` plugin:
 
-1. **Avoid Calling `player.reset()`:** If possible, avoid using `player.reset()` unless absolutely necessary. In many
-   cases, it may not be required.
+1. :warning: **Avoid Calling `player.reset()`:** If possible, avoid using `player.reset()` unless absolutely necessary.
```

Comment:
Could we use the proper emoji instead ?

Reply:
Sure, I've settled for 🚨 since the default warning emojii is not very eye-popping.